openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -463,15 +462,13 @@ public void jckConfigurationForMultijvmWithNoAgent(StfCoreExtension test) throws
 			throw new StfException("Unable to get a free port");
 		}
 		
-		// If Windows is specified for the property jck.env.testPlatform.os then 
-		// different properties are used for some settings - e.g. parameter
-		// jck.env.runtime.testExecute.cmdAsFile is used for the path to java rather than
-		// jck.env.runtime.testExecute.cmdAsFile.
-		// If "Current system" is specified for jck.env.testPlatform.os then the non Windows
-		// values appear to be used and execute the tests as normal.
-		// "Current system" should also be chosen for the property in the JCK Configuration editor
-		// for the template .jti files.
-		testPlatform = "Current system";
+		// Set the operating system as 'Windows' for Windows and 'other' for all other operating
+		// systems.
+		// If 'other' is specified when executing on Windows, then Windows specific settings such
+		// as systemRoot are rejected, and the JCK harness assumes that DISPLAY is required for
+		// GUI tests.
+		// Otherwise 'other' is required because the JCK harness has no inherent knowledge of AIX and
+		// zOS.
```

Comment:
hanging widow (one word on a line) words are a bit annoying to the eye, but no biggy.

Reply:
Moved onto preceding line.  Just happened to be how big my editor window was at the time!